app/services/idv/financials_step.rb

```diff
@@ -29,6 +29,10 @@ def vendor_errors
       vendor_validator.errors if form_valid?
     end
 
+    def vendor_reasons
+      vendor_validator.reasons if form_valid?
```

Comment:
Also, errors are a subset of reasons, not the other way around. Reasons can be positive (indicating why something passed).